docs/index.rst

```diff
@@ -216,3 +216,72 @@ In dask-distributed, a ``Worker`` is a Python object and node in a dask
 computations. ``Jobs`` are resources submitted to, and managed by, the job
 queueing system (e.g. PBS, SGE, etc.). In dask-jobqueue, a single ``Job`` may
 include one or more ``Workers``.
+
+How to debug
+------------
+
+Dask jobqueue has been developed and tested by several contributors, each of
+us having a given HPC system setup to work on: a job scheduler in a given
+version running on a given OS. Thus, in some specific case, it might not work
+out of the box on your system. This section provides some hints to help you
+sort what may be going wrong.
+
+Checking job script
+~~~~~~~~~~~~~~~~~~~
+
+Often, the first thing to do is checking if the job script generated to submit
+jobs launching dask workers looks like your usual submission script, in
+particular the header containing ``#PBS``, ``#SBATCH`` or other directives.
+This can be done easily once you've created a cluster object:
+
+.. code-block:: python
+
+   print(cluster.job_script())
+
+Next move if you don't see any problem is to actually try to run this script,
```

Comment:
If everything in job script appears correct, the next step is to try to submit a test job using the script. You can simply copy and paste...